internal/pkg/agent/cmd/enroll_cmd_test.go

```diff
@@ -216,16 +228,24 @@ func TestEnroll(t *testing.T) {
 			require.NoError(t, err)
 
 			streams, _, _, _ := cli.NewTestingIOStreams()
-			err = cmd.Execute(context.Background(), streams)
-			require.NoError(t, err)
-
-			require.True(t, store.Called)
-
+			ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+			defer cancel()
+			err = cmd.Execute(ctx, streams)
+			if err != nil &&
```

Comment:
is the error always returned here ?

Reply:
I'm not quite sure what you mean.

If there is an error it should be returned.